articles/azure-functions/functions-manually-run-non-http.md

```diff
@@ -65,6 +65,9 @@ Open Postman and follow these steps:
 
     :::image type="content" source="./media/functions-manually-run-non-http/functions-manually-run-non-http-body.png" alt-text="Postman body settings." border="true":::
 
+> [!NOTE]
+> If you don't want to pass data into the function, you must still pass an empty dictionary `{}` as the body of the POST request.
+
 1. Select **Send**.
```

Comment:
```suggestion
   > [!NOTE]
   > If you don't want to pass data into the function, you must still pass an empty dictionary `{}` as the body of the POST request.

1. Select **Send**.
```
adding indentation to prevent automatic numbering reset